support-services/src/test/scala/com/gu/support/pricing/PriceSummaryServiceSpec.scala

```diff
@@ -30,8 +32,25 @@ class PriceSummaryServiceSpec extends AsyncFlatSpec with Matchers {
     val digitalPack = service.getPrices(DigitalPack, List(discountPromoCode))
     digitalPack(UK)(NoFulfilmentOptions)(NoProductOptions)(Monthly)(GBP).price shouldBe 11.99
     digitalPack(UK)(NoFulfilmentOptions)(NoProductOptions)(Monthly)(GBP).promotions.head.discountedPrice shouldBe Some(8.39)
-    digitalPack(UK)(NoFulfilmentOptions)(NoProductOptions)(Annual)(GBP).price shouldBe 99
-    digitalPack(UK)(NoFulfilmentOptions)(NoProductOptions)(Annual)(GBP).promotions.head.discountedPrice shouldBe Some(91.57)
+    digitalPack(UK)(NoFulfilmentOptions)(NoProductOptions)(Annual)(GBP).price shouldBe 119
```

Comment:
The catalog-prod.json hadn't been updated since the price rise

Reply:
oops, I wonder if we should make it auto update when you run tests with credentials?

Reply:
I don't think it really matters, we're just testing that we can read the catalog.json successfully not that the prices are up to date.